query/src/test/scala/filodb/query/exec/RemoteMetadataExecSpec.scala

```diff
@@ -167,16 +167,10 @@ class RemoteMetadataExecSpec extends AnyFunSpec with Matchers with ScalaFutures
 
     val resp = exec.execute(memStore, querySession).runToFuture.futureValue
     val result = (resp: @unchecked) match {
-      case QueryResult(id, _, response, _, _, _) => {
-        val rv = response(0)
-        rv.rows.size shouldEqual 0
-        rv.rows.map { row =>
-          val record = row.asInstanceOf[BinaryRecordRowReader]
-          rv.asInstanceOf[SerializedRangeVector].schema.toStringPairs(record.recordBase, record.recordOffset)
-        }
```

Comment:
Sketched out by this-- @sherali42 , do we need the result to contain an empty RV?